packages/components/addon/components/hds/table/index.hbs

```diff
@@ -54,8 +57,9 @@
       {{#each (sort-by this.getSortCriteria @model) key=this.identityKey as |record|}}
         {{yield
           (hash
-            Tr=(component "hds/table/tr")
+            Tr=(component "hds/table/tr" isSelectable=@isSelectable)
             Th=(component "hds/table/th" scope="row")
+            ThSelectable=(component "hds/table/th-selectable" scope="row")
```

Comment:
```suggestion
```

Reply:
Thanks I alrady figured out I need to remove these as I was working on it.